straxen/plugins/events/multi_scatter.py

```diff
@@ -55,37 +39,71 @@ def infer_dtype(self):
     max_drift_length = straxen.URLConfig(
         default=straxen.tpc_z, infer_type=False,
         help='Total length of the TPC from the bottom of gate to the '
-             'top of cathode wires [cm]', )
+             'top of cathode wires [cm]')
+
+    ms_window_fac = straxen.URLConfig(
+        default=1.01, type=(int, float),
+        help='Max drift time window to look for peaks in multiple scatter events'
+    )
+
+    def infer_dtype(self):
+        dtype = strax.time_fields + [
```

Comment:
Would you also add comments on these variables?

Reply:
Done in https://github.com/XENONnT/straxen/pull/1162/commits/ec47dd20dbf7fec29ae742f5672f7345410c0276